text/src/autogluon/text/automm/models/categorical_transformer.py

```diff
@@ -0,0 +1,275 @@
+import torch
+from torch import nn
+from torch import Tensor
+from typing import Any, Callable, Dict, List, Optional, Tuple, Type, Union, cast
+from ..constants import CATEGORICAL, LABEL, LOGITS, FEATURES
+from .ft_transformer import _TokenInitialization, CLSToken, Transformer
+
+
+class CategoricalFeatureTokenizer(nn.Module):
+    """
+        Transforms categorical features to tokens (embeddings).
+    """
+
+    category_offsets: Tensor
```

Comment:
We may remove this definition here.

Reply:
We move this definition into `__init__`.